src/components/QuickOpenModal.js

```diff
@@ -115,7 +115,7 @@ export class QuickOpenModal extends Component<Props, State> {
       const results = this.props.sources;
       return this.setState({ results });
     }
-    if (this.isGotoSourceQuery()) {
+    if (this.isGotoSourceQuery() || this.isSourcesQuery()) {
```

Comment:
hmm not sure what this does. can you add a unit test?

Reply:
Sure, this causes the error emoji to disappear if the source has a `:`, but it is only half of the issue - the other half is that when the input is in `SourcesQuery` mode it does not consider the location suffix.